### PR TITLE
API-3828

### PR DESCRIPTION
**Pull request by cjrowe**

### CODE REVIEW

File: test/it/uk/gov/hmrc/thirdpartyapplication/repository/ApplicationRepositorySpec.scala [hunk 1]
```diff
@@ -102,13 +103,46 @@ class ApplicationRepositorySpec extends UnitSpec with MongoSpecSupport
...
+    }
+
+    "set lastAccess property on applications that do not have it" in {
+      val application1Id = UUID.randomUUID()
+      val application2Id = UUID.randomUUID()
+      val dateToSet: DateTime = DateTime.now
+
+      await(applicationWithoutLastAccessDate(application1Id))
+      await(applicationWithoutLastAccessDate(application2Id))
+      await(applicationRepository.save(anApplicationData(UUID.randomUUID()))) // Application with lastAccess date
```
Comment (FluentSynergyDW) [inline test/it/uk/gov/hmrc/thirdpartyapplication/repository/ApplicationRepositorySpec.scala]: Is it worth having a third application that does have a last access date and can be checked for being unchanged?

Comment (cjrowe) [reply test/it/uk/gov/hmrc/thirdpartyapplication/repository/ApplicationRepositorySpec.scala]: Do have one - the third item there (line 133) :-)

**Comment by cjrowe** [comment]
> We have a story to check whether the scheduled jobs in third-party-application are running after Luis raised this a little while ago. This work involves the scheduled jobs so is it possible to verify that the existing jobs are running as part of testing this work?

Spoke to Luis about this and should have fixed the jobs not running - no test in there to confirm it though...

File: test/it/uk/gov/hmrc/thirdpartyapplication/repository/ApplicationRepositorySpec.scala (hunk 1 above)
Comment (FluentSynergyDW) [reply test/it/uk/gov/hmrc/thirdpartyapplication/repository/ApplicationRepositorySpec.scala]: Oh yeah! But we don't check for it being unchanged, we only check the count, which is not quite the same.

**Pull request merged**